### PR TITLE
[PD 2835] responsive css cleanup

### PR DESCRIPTION
**Pull request by mwarnsleyDE**
Task was to cleanup some of the responsiveness for the css and cleanup some of the code.

To Test:

Pull down the repository. You can shrink the screen size down to mobile and test the tabs and the primary dimensions to see how they work with mobile.

### CODE REVIEW

**Comment by koonsies** [comment]
Has Jason Sole seen the menu location/tab change? It's a pretty big step, so I wanted to make sure it's what he's expecting to see.

**Comment by koonsies** [comment]
Currently, you can click on the text in the bottom menu to change the chart type, but when you hover over it, it displays a text "I" symbol. Not a huge deal, but since the mobile view is currently large enough to become active on a small laptop screen, it should probably be addressed.

**Comment by mwarnsleyDE** [comment]
We might have to separate this ticket out to anything else that might need to be done. If anyone has any suggestions or can take a look at the maps and legend as far as it being responsive please feel free too because that's about the best I can come up with right now. I hid the legend below a certain viewport based off of the fact that it didn't look good at such a small screen size

**Comment by koonsies** [comment]
Alright. I will begin review. This is the "as-is" state of this ticket? No further commits?

**Comment by mwarnsleyDE** [comment]
Correct, I don't have any further commits for this ticket

File: gulp/src/analytics/components/SideBar/MobileDimensionList.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  selectTab() {
+    const {active, close} = this.props;
+    active();
+    close();
+  }
+  render() {
+    const {dimension, selected} = this.props;
+    return (
+      <li onClick={this.selectTab.bind(this)} className={selected === dimension.value ? 'side-dimension active-main' : 'side-dimension'}>
+         <span className="side-circle-btn"></span><span className="side-dimension-title">{dimension.display}</span>
```
Comment (mwarnsleyDE) [inline gulp/src/analytics/components/SideBar/MobileDimensionList.js]: For this one, there aren't hover effects when on a mobile device if you're referring to all of this on the mobile screen

Comment (koonsies) [inline gulp/src/analytics/components/SideBar/MobileDimensionList.js]: Yeah, you can't hover on a mobile device, but the current responsive settings have it swapping to a mobile view as only a little under 100% utilization of a laptop screen, therefore it's still likely that a computer would receive the mobile view.

**Comment by mwarnsleyDE** [comment]
Fixed

**Comment by mwarnsleyDE** [comment]
Hey, I can't seem to tell why these test are failing for some reason. Could be the merge I did but I'm not too sure.

**Comment by tdphillips** [comment]
I just ran tests locally using this branch. It's probably a temporary thing.

**Comment by koonsies** [comment]
I pulled the latest changes down, but the CSS is broken for me. I'm not sure if anyone else is having this issue.
screen shot 2017-01-10 at 5 01 16 pm

**Pull request merged**